WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderCommentAdapter.java

```diff
@@ -279,16 +280,32 @@ public void onClick(View view) {
         if (mIsLoggedOutReader) {
             commentHolder.txtReply.setVisibility(View.GONE);
             commentHolder.imgReply.setVisibility(View.GONE);
-        } else if (mReplyListener != null) {
-            // tapping reply icon tells activity to show reply box
-            View.OnClickListener replyClickListener = new View.OnClickListener() {
-                @Override
-                public void onClick(View v) {
-                    mReplyListener.onRequestReply(comment.commentId);
-                }
-            };
-            commentHolder.txtReply.setOnClickListener(replyClickListener);
-            commentHolder.imgReply.setOnClickListener(replyClickListener);
+        } else {
+            if (mReplyListener != null) {
+                // tapping reply icon tells activity to show reply box
+                View.OnClickListener replyClickListener = new View.OnClickListener() {
+                    @Override
+                    public void onClick(View v) {
+                        mReplyListener.onRequestReply(comment.commentId);
+                    }
+                };
+                commentHolder.txtReply.setOnClickListener(replyClickListener);
+                commentHolder.imgReply.setOnClickListener(replyClickListener);
+            }
+
+            if (mDoLikeCommentId != 0 && mDoLikeCommentId == comment.commentId) {
```

Comment:
Ah, I see what you mean. Note that it's not the user that is expected to scroll to the comment. Rather, the parent activity asks the list view to scroll there (same way scrolling works when opened via the notifications) and the scrolling happens automatically. If for any reason the specific list row doesn't render (and so it doesn't become visible to the user) then an error must be assumed and the action itself not performed.

I'd recommend to leave it as it is. To change it to perform the action in a "api" style is not trivial. We can always address it in a future PR. WDYT?